code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -23,6 +23,20 @@ God bless America.
 #define DEEPFRYER_COOKTIME 60
 #define DEEPFRYER_BURNTIME 120
 
+GLOBAL_LIST_INIT(deepfry_blacklisted_items, typecacheof(list(
+	/obj/item/screwdriver,
+	/obj/item/crowbar,
+	/obj/item/wrench,
+	/obj/item/wirecutters,
+	/obj/item/multitool,
+	/obj/item/weldingtool,
```

Comment:
would it be possible to make a separate list that only the deep fryer checks, and move the tools here to that list? the only reason they're on this list is so that you don't accidentally deep fry them while trying to maintain/deconstruct a deep fryer, which doesn't seem relevant for cooking oil being applied by a sprayer.

Reply:
Good idea, I've moved the tools off into a different list, so now they should be fryable again.